aospy/test/data/objects/examples.py

```diff
@@ -1,6 +1,7 @@
-from datetime import datetime
 import os
 
+from cftime import DatetimeNoLeap
```

Comment:
This means we'll need to add `cftime` as a required dependency, right?

Reply:
Good catch.  `cftime` is a required dependency of `netcdf4` now, but we can add it to our `setup.py` to be explicit and also maintain support for older versions of `netcdf4`.